front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -46,7 +46,7 @@ export const ParticipationMethodPicker = ({
   handleParticipationMethodOnChange,
 }: Props) => {
   const chooseParticipationMethod = () => {
-    if (!isNilOrError(phase)) {
+    if (!isNilOrError(phase) && phase.data) {
```

Comment:
You can just check for `phase?.data` here, no? We are trying to get rid of the `isNilOrError` checks.